integration/helpers/deno-template/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "remix-template-deno",
+  "private": true,
+  "sideEffects": false,
+  "dependencies": {
+    "@remix-run/deno": "*",
+    "@remix-run/react": "*",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0"
```

Just like in the `remix-deno` template, I think this should still be v17 to be consistent with all other templates we have.

```suggestion
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
```